docs/program-setup/bare_metal/mbed2_porting.md

```diff
@@ -0,0 +1,125 @@
+# Porting an Mbed OS 2 target to Mbed OS 6 bare metal
+
+The Mbed OS bare metal profile removes the RTOS and provides fewer features compared to Mbed OS 6. This profile offers the same functionality as Mbed OS 2 and allows targets to access features that we have added to more recent versions of Mbed OS. This document describes how to configure an Mbed OS 2 target for bare metal and validate the port.
```

Comment:
functionality or functionalities?

Reply:
Probably "functionality"

Reply:
So Mbed 2 provides one functionality? What is it?

Reply:
Functionality is a collective term. 

"The range of operations that can be run on a computer or other electronic system: new software with additional functionality."